src/goto-analyzer/build_analyzer.cpp

```diff
@@ -0,0 +1,164 @@
+/*******************************************************************\
+
+Module: Goto-Analyser Command Line Option Processing
```

Comment:
⛏️  (multiple locations)
```suggestion
Module: Goto-Analyzer Command Line Option Processing
```

Reply:
Done.